buildkite/bazelci.py

```diff
@@ -824,7 +824,7 @@ def common_flags(bep_file, platform):
             "--experimental_multi_threaded_digest"]
 
 
-def rbe_flags(accept_cached):
+def rbe_flags(original_flags, accept_cached):
```

Comment:
the original_flags contain the user specified flags in the yml file I assume?

Reply:
Yes.